layouts/shortcodes/studio-hub.html

```
@@ -0,0 +1,259 @@
+                Studio →{{ else }}How Altinn Studio works →{{ end }}</a></div>
+    </div>
+    <section class="authz-section authz-wrap">
+        <div class="authz-section-head">
+            <div>
+                <h2>{{ if $nb }}Start med rollen din{{ else }}Start with your role{{ end }}</h2>
+                <p>{{ if $nb }}Vi leder deg til riktig løp og riktig detaljnivå.{{ else }}We will guide you to the right
+                    journey and level of detail.{{ end }}</p>
+            </div>
+        </div>
+        <div class="authz-grid">
+            <a class="authz-card authz-role" href="{{ "altinn-studio/v9/getting-started/serviceowner" | relLangURL }}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)